Notifier/UI/Windows/NotifierTrayIcon.cs

```diff
@@ -135,6 +128,7 @@ void ShowBalloonTip()
                 }
             }
 
+            // TODO: @harrwiss, tooltipText is a constant, so it's size is already known.
             if (tooltipText.Length > 64)
```

Comment:
Yes, but who knows and somebody changes the text in the constant later. 
If it is larger than 64 chars then the effect ist afair that windows does not show a tooltip at all - I had to troubleshoot this already and want to avoid it in future 😄

Reply:
A compile time check could then be better... but I don't know how to implement it as of now, to be honest.